orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/gce/GceDeployStagePreProcessor.groovy

```diff
@@ -113,7 +115,8 @@ class GceDeployStagePreProcessor implements DeployStagePreProcessor  {
   }
 
   private static boolean shouldPinSourceServerGroup(String strategy) {
-    return strategy == "redblack" // || strategy == "rollingredblack" TODO(jacobkiefer): Insert if/when RRB is implemented for GCE.
+    return (Strategy.fromStrategy(strategy) != Strategy.RED_BLACK)
+    // || Strategy.fromStrategy(strategy) == Strategy.ROLLING_RED_BLACK TODO(jacobkiefer): Insert if/when RRB is implemented for GCE.
```

Comment:
should this line be commented out?

Reply:
It was like this, I wanted to keep it around for whoever commented it out